src/restore.ts

```diff
@@ -19,9 +19,8 @@ async function restore(ccacheVariant : string) : Promise<void> {
     restoreKeys: core.getInput("restore-keys").split("\n").map(s => s.trim()).filter(x => x !== "")
   };
 
-  const keyPrefix = ccacheVariant + "-";
-  const primaryKey = inputs.primaryKey ? keyPrefix + inputs.primaryKey + "-" : keyPrefix;
-  const restoreKeys = inputs.restoreKeys.map(k => keyPrefix + k + "-")
+  const primaryKey = inputs.primaryKey ? ccacheVariant + "-" + inputs.primaryKey : ccacheVariant;
+  const restoreKeys = inputs.restoreKeys.map(k => ccacheVariant + "-" + k)
```

Comment:
@compnerd 

the concrete line I talk about. I understand that before the change the restore key is:

"ccache-{k}-"

after

"ccache-{k}"

Reply:
Right, so, I made that change because I changed the save key. If that change is not needed, I'm totally happy to undo it. Maybe I didn't fully grok how the save/restore keys work.